### PR TITLE
ZTE-SH-CN: downward-api-volume-expose-pod-information.md

### PR DESCRIPTION
**Pull request by houjun41544**
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.8 Features: set Milestone to `1.8` and Base Branch to `release-1.8`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5450)

### CODE REVIEW

File: cn/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+```shell
+/# exit
+```
+
+## 存储容器字段
+
+前面的练习中，你将Pod字段保存到DownwardAPIVolumeFile中。接下来这个练习，你将存储容器字段。这里是包含一个容器的pod的配置文件：
+
+{% include code.html language="yaml" file="dapi-volume-resources.yaml" ghlink="/docs/tasks/inject-data-application/dapi-volume-resources.yaml" %}
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md]: copy a dapi-volume-resources.yaml to /cn/docs/tasks/inject-data-application/dapi-volume-resources.yaml

File: cn/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+* [环境变量](/docs/tasks/configure-pod-container/environment-variable-expose-pod-information/)
+* DownwardAPIVolumeFiles
+
+这两种呈现Pod和Container字段的方式都称为*Downward API*。
+
+## 存储Pod字段
+
+在这个练习中，你将创建一个包含一个容器的pod。这是该pod的配置文件：
+
+{% include code.html language="yaml" file="dapi-volume.yaml" ghlink="/docs/tasks/inject-data-application/dapi-volume.yaml" %}
```
Comment (zhangxiaoyu-zidif) [inline cn/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md]: copy a dapi-volume-resources.yaml to /cn/docs/tasks/inject-data-application/dapi-volume-resources.yaml
and modify the path:/docs/tasks/inject-data-application/ to /cn/docs/tasks/inject-data-application/

Comment (zhangxiaoyu-zidif) [reply cn/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md]: lack of  dapi-volume-resources.yaml results in netlify/deploy failed.

**Comment by zhangxiaoyu-zidif** [comment]
image
@houjun41544 
please check the encode of you files. Thanks.

File: cn/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+
+在这个练习中，你将创建一个包含一个容器的pod。这是该pod的配置文件：
+
+{% include code.html language="yaml" file="dapi-volume.yaml" ghlink="/cn/docs/tasks/inject-data-application/dapi-volume.yaml" %}
+
+在配置文件中，你可以看到Pod有一个`downwardAPI`类型的Volume，并且挂载到容器中的`/etc`。
+
+查看`downwardAPI`下面的`items`数组。每个数组元素都是一个[DownwardAPIVolumeFile](/docs/resources-reference/{{page.version}}/#downwardapivolumefile-v1-core)。
+第一个元素指定Pod的`metadata.labels`字段的值保存在名为`labels`的文件中。
+第二个元素自定Pod的`annotations`字段的值保存在名为`annotations`的文件中。
```
Comment (tengqm) [inline cn/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md]: 自定->指示

File: cn/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+```
+你可以使用同样的命令查看`cpu_request`, `mem_limit` 和`mem_request` 文件.
+
+{% endcapture %}
+
+{% capture discussion %}
+
+## Capabilities of the Downward API
+
+下面这些信息可以通过environment variables和DownwardAPIVolumeFiles提供给容器：
```
Comment (tengqm) [inline cn/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md]: environment variables要翻译

File: cn/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+
+## 投射密钥到指定路径并且指定文件权限
+
+你可以将密钥投射到指定路径并且指定每个文件的访问权限。更多信息，请参阅[Secrets](/docs/concepts/configuration/secret/).
+
+## Downward API的动机
+
+对于容器来说，有时候拥有自己的信息是很有用的，可避免与Kubernetes过度耦合。Downward API使得容器使用自己或者集群的信息，而不必通过Kubernetes client或API server。
+
+一个例子是有一个应用采用一个特殊周知的环境变量来保存一个唯一标识。一种可能是给应用增加处理层，但这样是冗余和易出错的，而且它违反了低耦合的目标。
```
Comment (tengqm) [inline cn/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md]: 特殊周知的？

File: cn/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+1
+33554432
+67108864
+```
+
+{% endcapture %}
+
+{% capture whatsnext %}
+
+* [Defining Environment Variables for a Container](/docs/tasks/configure-pod-container/define-environment-variable-container/)
```
Comment (tengqm) [inline cn/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md]: 链接的文字要翻译

**Comment by houjun41544** [comment]
I have fixed all errors you mentioned,please review again,thinks. @tengqm

File: cn/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md [hunk 7]
```diff
@@ -0,0 +1,209 @@
...
+* node的IP
+* Pod的name
+* Pod的namespace
+* Pod的IP address
+* Pod的service account name
+* Pod的UID
+* Container的CPU limit
+* Container的CPU request
+* Container的memory limit
+* Container的memory request
```
Comment (tengqm) [inline cn/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md]: 以上列表中很多项可以翻译并且不耽误理解。例如：

* 节点名称
* 节点IP
* Pod名称
* Pod名字空间
* Pod IP地址
* Pod 服务帐号名称
* 容器的CPU约束（limit）
* 容器的CPU请求（request）
...

File: cn/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md [hunk 8]
```diff
@@ -0,0 +1,209 @@
...
+
+此外，以下信息可通过DownwardAPIVolumeFiles获得：
+
+* Pod的labels
+* Pod的annotations
```
Comment (tengqm) [inline cn/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md]: 同样，这里的labels、annotations都可以翻译

File: cn/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md [hunk 9]
```diff
@@ -0,0 +1,209 @@
...
+**Note:** 如果容器未指定CPU和memory limits，则Downward API默认为节点可分配值。
+{: .note}
+
+## 投射密钥到指定路径并且指定文件权限
+
+你可以将密钥投射到指定路径并且指定每个文件的访问权限。更多信息，请参阅[Secrets](/docs/concepts/configuration/secret/).
+
+## Downward API的动机
+
+对于容器来说，有时候拥有自己的信息是很有用的，可避免与Kubernetes过度耦合。Downward API使得容器使用自己或者集群的信息，而不必通过Kubernetes client或API server。
```
Comment (tengqm) [inline cn/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md]: ... 而不必通过Kubernetes客户端或者API服务器。

**Comment by houjun41544** [comment]
@tengqm Thinks for your advice. I have mofied.

**Pull request merged**